logrotate.d/nginx

```diff
@@ -0,0 +1,15 @@
+/var/log/nginx/*.log {
+        daily
+        missingok
+        rotate 52
```

Comment:
I understand it's the default, but I'm still asking the question- do we as hashbang need that many days?